packages/vue/src/components/molecules/SfReview/SfReview.js

```diff
@@ -0,0 +1,112 @@
+import { SfRating } from "@storefront-ui/vue";
+
+export default {
+  name: "SfReview",
+  props: {
+    /**
+     * Author of the review
+     */
+    author: {
+      type: String,
+      required: true
+    },
+    /**
+     * Date of the review
+     */
+    date: {
+      type: String,
+      required: true
+    },
+    /**
+     * Message from the reviewer
+     */
+    message: {
+      type: String,
+      required: true
```

Comment:
```suggestion
      default: ""
```
default is better, please use it always 👍